doc/playbook.rst

```diff
@@ -1,108 +1,267 @@
+* ``nr_rows`` - number of rows to conduct the experiment over. This is the number of aggressor rows.
+  Victim rows will be 2 times fewer than this number. For example, to perform hammering for 32 victim rows, use 34 as the parameter value
+* ``read_count_step`` - this is how much to increment the hammer count between multiple tests for the same row.
+  This is the number of hammers on single side (total number of hammers on both sides is 2x this value)
+* ``iters_per_row`` - number of times the hammer count is incremented for each row
+
+The results are a series of histograms with appropriate labeling.
+
+Example :ref:`configurations` for this test were provided as ``configs/example_hammer_*.cfg`` files.
+Some of them require a significant amount o memory declared as `payload memory`.
+To execute a minimalistic example from within litex-rowhammer-tester repo enter:
+
+.. code-block:: console
+
+  source venv/bin/activate
+  cd rowhammer_tester/scripts/playbook/
+  python playbook.py configs/example_hammer_minimal.cfg
+
+Expected output:
+
+.. code-block:: console
+
+  Progress: [========================================] 3072 / 3072 
+  Generating payload:
+    tRAS = 5
+    tRP = 3
+    tREFI = 782
+    tRFC = 32
+    Repeatable unit: 186
+    Repetitions: 93
+    Payload size =  0.04KB /  1.00KB
+    Payload per-row toggle count = 0.010K  x2 rows
+    Payload refreshes (if enabled) = 10 (disabled)
+    Expected execution time = 1263 cycles = 0.013 ms
+  
+  Transferring the payload ...
+  
+  Executing ...
+  Time taken: 0.647 ms
+  
+  Progress: [============                            ]  323 / 1024 (Errors: 320) 
+  ...
+
+Configurations
+--------------
+
+Test configuration files are represented as python dictionaries. An example:
+
+.. code-block:: python
+
+    {
+        "inversion_divisor" : 2,
+        "inversion_mask" : "0b10",
+        "payload_generator" : "RowListPayloadGenerator",
+        "payload_generator_config" : {
+            "row_mapping" : "TypeARowMapping",
+            "row_generator" : "EvenRowGenerator",
+            "read_count" : 27,
+            "max_iteration" : 10,
+            "verbose" : true,
+            "refresh" : false,
+            "fill_local" : true,
+            "row_generator_config" : {
+                "nr_rows" : 10,
+                "max_row" : 64
+            }
+        }
+    }
+
+Different parameters are supported:
+
+* ``payload_generator`` - name of the :ref:`payload generator class` to use
+* ``row_pattern`` - pattern that will be stored in rows
+* inversion_divisor and inversion_mask - controls which rows get the inverted pattern
+  described in :ref:`inversion`
+* ``payload_generator_config`` - these parameters are specific for :ref:`payload generator class` used
+
+Inversion
+*********
+
+If needed, some of the tested rows can be `inverted` - all of their bits flipped to the opposite value.
```

Comment:
I would explicitly mention that "the data pattern for some of the selected rows can be bitwise-inverted".

Reply:
Applied suggestion